package.json

```diff
@@ -31,6 +31,7 @@
     "gulp-start": "~1",
     "gulp-uglify": "~2",
     "gulp-util": "~3",
+    "gulp-watch": "~4",
```

Comment:
It is not necessary for the core - is just a way to workaround [this problem](https://forums.docker.com/t/docker-compose-volumes-gulp-watch-100-cpu-usage/10192).